### ISSUE TITLE
Prevent SSHD startup errors appearing in docker logs

### ISSUE THREAD

**Issue by jdeathe** [jdeathe/centos-ssh#95]
```
...
sshd stdout | Could not load host key: /etc/ssh/ssh_host_rsa_key
sshd stdout | Could not load host key: /etc/ssh/ssh_host_dsa_key
Server listening on 0.0.0.0 port 22.
Server listening on :: port 22.
sshd_bootstrap stdout | Initialise SSH...
```

Issue occurs because Supervisord does not wait for completion of one process before starting another with a lower priority.

Will resolve issue raised here too: https://github.com/jdeathe/centos-ssh/pull/3
